charts/castai-pod-node-lifecycle/templates/pod-disrubtion-budget.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   minAvailable: 1
   selector:
     matchLabels:
-      app: {{ include "pod-node-lifecycle.fullname" . }}
+      app.kubernetes.io/name: {{ include "pod-node-lifecycle.fullname" . }}
```

Comment:
not sure what's the reasoning for this

Reply:
Our pods don't use app: castai-pod-node-lifecycle label so this PDB hasn't worked in awhile.